src/libraries/System.Private.CoreLib/src/System/Guid.cs

```diff
@@ -1219,8 +1219,8 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
                         Vector128<byte> nibbles = Vector128.ShiftRightLogical(srcVec.AsUInt64(), 4).AsByte();
                         Vector128<byte> lowNibbles = UnpackLow(nibbles, srcVec) & Vector128.Create((byte)0xF);
                         Vector128<byte> highNibbles = UnpackHigh(nibbles, srcVec) & Vector128.Create((byte)0xF);
-                        (Vector128<ushort> v0, Vector128<ushort> v1) = Vector128.Widen(Shuffle(hexMap, lowNibbles));
```

Comment:
Can you revert changes in this file (Guid.cs) since I overwrote them in https://github.com/dotnet/runtime/pull/81666 so we can avoid a conflict

Reply:
Reverted